build.gradle.kts

```
@@ -1,6 +1,6 @@
 plugins {
   id("uk.gov.justice.hmpps.gradle-spring-boot") version "5.7.0"
-  kotlin("plugin.spring") version "1.8.22"
+  kotlin("plugin.spring") version "1.9.20"
```

```suggestion
  kotlin("plugin.spring") version "1.9.10"
```
since we haven't upgraded the plugin to 1.9.20 yet (hadn't realised it had been released - latest according to https://github.com/JetBrains/kotlin/releases is 1.9.10 right now